src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -834,12 +911,12 @@ static int do_unmap(const std::string &devpath)
   return r;
 }
 
-static int parse_imgpath(const std::string &imgpath, Config *cfg)
-{
+static int parse_imgpath(const std::string &imgpath, Config *cfg,
+                         std::ostream *err_msg) {
```

Comment:
NIT:tab

Reply:
I am not sure I see what is wrong with indentation? Both spaces and tabs are allowed by the guide and I prefer spaces.